packages/reports/addon/templates/components/editable-label.hbs

```diff
@@ -1,23 +1,26 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#if isEditingValue}}
-  {{input
-    classNames="editable-label__input"
-    value=_localValue
-    size=_inputSize
-    maxlength=150
-    enter=(action "blurLabelInput")
-    focus-out=(action "editComplete")
-  }}
-{{else}}
-  <span
-    class="clickable editable-label__value"
-    {{action (pipe (action "resetValue") (toggle "isEditingValue" this) (action "highlightLabelInput"))}}
-    role="button"
-  >
-    {{value}}
-    <span>
-      {{navi-icon "pencil" class="editable-label__icon"}}
-      {{ember-tooltip text="Rename" popperContainer="body" effect="none"}}
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="editable-label" ...attributes>
+  {{#if this.isEditingValue}}
+    <Input
+      @classNames="editable-label__input"
+      @value={{this._localValue}}
+      @size={{this._inputSize}}
+      @maxlength={{150}}
+      @enter={{fn this.blurLabelInput}}
+      @focus-out={{fn this.editComplete}}
```

Comment:
```suggestion
      @enter={{this.blurLabelInput}}
      @focus-out={{this.editComplete}}
```

Do we need the `fn`?

Reply:
Yes, it's a bug with the <Input /> component but I didn't want to refactor to stop using it